### PR TITLE
fix: Vercel deployment with PWA module example

### LINKED ISSUE #1348: Not found static files in example of Vercel deployment with PWA module
## Summary

I tried to deploy Nuxt.js app with Vercel following [Service Worker with Nuxt PWA Module](https://nuxtjs.org/docs/2.x/deployment/vercel-deployment#service-worker-with-nuxt-pwa-module) page, but contents in `static` derictory were not found.

Contents in `static` directory such as `favicon.ico` are deployed not under `/_nuxt/static` directory but at document root by vercel-builder.

So, fixing `vercel.json` like below works fine.

```diff
 "routes": [
   { "src": "/_nuxt/.+", "headers": { "Cache-Control": "max-age=31557600" } },
   {
     "src": "/sw.js",
-    "dest": "/_nuxt/static/sw.js",
     "headers": {
       "cache-control": "public, max-age=43200, immutable",
       "Service-Worker-Allowed": "/"
     }
   },
   { "src": "/(.*)", "dest": "/" }
 ]
```

And according to [Vercel docs](https://vercel.com/docs/configuration?query=version#project/routes), `routes` property is deprecated. I think `headers` property should be used.

I'll create a PR soon.

## Reproduction repository

- following the current documentation (not found static files)
  - [repository](https://github.com/nokazn/vercel-nuxtjs-example)
  - [deployment](https://vercel-nuxtjs-example-ef73a0xbw-nokazn.vercel.app/)
- modified `routes` to `header` in `vercel.json`
  - [repository](https://github.com/nokazn/vercel-nuxtjs-example/tree/pwa)
  - [deployment](https://vercel-nuxtjs-example-hcojdpfea-nokazn.vercel.app/)

### PR DESCRIPTION
**Pull request by nokazn**
fix #1348

### CODE REVIEW

**Comment by danielroe** [comment]
Good catch indeed!

However, likely we no longer require this at all as Vercel now auto inject correct configuration for Nuxt: see https://github.com/vercel/vercel/blob/e2106d12f605a9bdaa2ebef54f182ef150deecb9/packages/frameworks/src/frameworks.ts#L1280-L1298. (Alternatively, we can submit PRs there.)

**Comment by nokazn** [comment]
@danielroe Yes, It works without `routes` property in `vercel.json`! ([deployment](https://vercel-nuxtjs-example-6z8q38vye-nokazn.vercel.app/))
Then, `routes` is not needed and `package.json` in `builds.config.serverFiles` is also unnecessary ([#21](https://github.com/nuxt/vercel-builder/issues/21#issuecomment-487963501)), so we can use the same file as described in [SSR with Vercel](https://nuxtjs.org/docs/2.x/deployment/vercel-deployment#ssr-with-vercel) section.
Should I remove [Service Worker with Nuxt PWA Module](https://nuxtjs.org/docs/2.x/deployment/vercel-deployment#service-worker-with-nuxt-pwa-module) section?

File: content/en/guides/deployment/vercel-deployment.md [hunk 1]
```diff
@@ -49,17 +49,29 @@ To avoid 404 for Service Workers, make sure to include `sw` to your routes setti
...
+        }
+      ]
     },
-    { "src": "/(.*)", "dest": "/" }
+    {
+      "source": "/sw.js",
+      "headers": [
+        {
+          "key": "Cache-Control",
+          "value": "public, max-age=43200, immutable"
```
Comment (danielroe) [inline content/en/guides/deployment/vercel-deployment.md:67]: While we're at it, I'd recommend instead
```suggestion
          "value": "public, max-age=0, must-revalidate"
```

**Review by inouetakuya — APPROVED** [review]
I checked https://github.com/nuxt/nuxtjs.org/blame/fd6ce6cb90c75a557e5541ac9bcbede36595ddad/content/en/guides/deployment/vercel-deployment.md#L40-L65 and this PR looks good to me! 👍 

Thank you!

**Review by danielroe — APPROVED** [review]

**Pull request merged**